### DISCUSSION TITLE
Certificates are not renewed automatically

### DISCUSSION THREAD

**Discussion by jmd9019** [digitalocean/Kubernetes-Starter-Kit-Developers#200] [Q&A]
Hi,
I am using nginx ingress and have deployed cert manager and ingress following this tutorial https://www.youtube.com/watch?v=MpovOI5eK58&list=RDCMUCaPX53JLxxSbwZz_Ra_cL0g&index=2
Its been almost 5-6 months since deployed I was expecting as per tutorial my certificates to be renewed automatically which didn't happen and my staging environment certificates are expired and need help with the renewal also where did I go wrong?
Also I need to fix this because I have deployed in same way to my production clusters also, so need help ASAP

**Comment by v-ctiutiu** [comment]
Follow up is on #201.

I don't think this is a bug on our end as stated in the above issue. Some possible situations/scenarios are detailed in #201.

@jmd9019 If the explanations from the above issue answer your question(s) please make sure to update it, or add the final resolution here in the discussions thread. Should help other users understand what really happened.

Thanks.
